dynawo/sources/Models/CPP/Components/ModelLoadRestorativeWithLimits/test/TestLoadRestorativeWithLimits.cpp

```diff
@@ -259,11 +263,6 @@ TEST(ModelsLoadRestorativeWithLimits, ModelLoadRestorativeWithLimitsContinuousAn
   ASSERT_NO_THROW(modelLoad->evalMode(0));
   ASSERT_NO_THROW(modelLoad->setGequations());
   ASSERT_NO_THROW(modelLoad->evalF(1, ALGEBRAIC_EQ));
-
-  y[1] = 0;
```

Comment:
why do we remove this?

Reply:
it is a wrong delete I did, it is back there